sonar-project.properties

```
@@ -14,8 +14,8 @@ sonar.sources=app/
 # Inclusions for test files.
 sonar.test.inclusions=**.test.**
 
-# Test coverage path
-sonar.javascript.lcov.reportPaths=/tests/coverage/lcov.info
+# Test coverage path in GitHub action
+sonar.javascript.lcov.reportPaths=**/tests/coverage/lcov.info
```

The docs use a relative path: https://docs.sonarcloud.io/enriching/test-coverage/javascript-typescript-test-coverage/

```suggestion
sonar.javascript.lcov.reportPaths=./tests/coverage/lcov.info
```